transport/src/main/java/io/netty/channel/FixedRecvByteBufAllocator.java

```diff
@@ -15,36 +15,23 @@
  */
 package io.netty.channel;
 
-import io.netty.buffer.ByteBuf;
-import io.netty.buffer.ByteBufAllocator;
-
-
 /**
  * The {@link RecvByteBufAllocator} that always yields the same buffer
  * size prediction.  This predictor ignores the feed back from the I/O thread.
  */
-public class FixedRecvByteBufAllocator implements RecvByteBufAllocator {
-
-    private static final class HandleImpl implements Handle {
+public class FixedRecvByteBufAllocator extends DefaultMaxMessagesRecvByteBufAllocator {
 
+    private final class HandleImpl extends MaxMessageHandle {
         private final int bufferSize;
 
-        HandleImpl(int bufferSize) {
+        public HandleImpl(int bufferSize) {
```

Comment:
The class is `private` however there is no reason why the visibility of this method should be restricted.